webextensions/manifest/permissions.json

```diff
@@ -85,18 +169,147 @@
             }
           }
         },
-        "activeTab": {
+        "clipboardWrite": {
+          "__compat": {
+            "support": {
+              "chrome": {
+                "version_added": true
+              },
+              "edge": {
+                "version_added": false
+              },
+              "firefox": {
+                "version_added": "51"
+              },
+              "firefox_android": {
+                "version_added": "51"
+              },
+              "opera": {
+                "version_added": true
+              }
+            }
+          }
+        },
+        "contentSettings": {
+          "__compat": {
+            "support": {
+              "chrome": {
+                "version_added": true
+              },
+              "edge": {
+                "version_added": false
+              },
+              "firefox": {
+                "version_added": false
+              },
+              "firefox_android": {
+                "version_added": false
+              },
+              "opera": {
+                "version_added": true
+              }
+            }
+          }
+        },
+        "contextMenu": {
           "__compat": {
             "support": {
               "chrome": {
                 "version_added": true
               },
               "edge": {
+                "version_added": true
+              },
+              "firefox": {
+                "notes": [
+                  "Available as an alias to the <code>"Menus"</code> permission."
```

Comment:
This is probably the line that causes the Travis failure, where the `"` needs to be escaped. Actually, it should be: `<code>menus</code>` (lowercase and not quoted).